pkg/cmd/generate-bazel-extra/main.go

```diff
@@ -41,8 +41,15 @@ var (
 )
 
 func runBuildozer(args []string) {
-	const buildozer = "_bazel/bin/external/com_github_bazelbuild_buildtools/buildozer/buildozer_/buildozer"
-	cmd := exec.Command(buildozer, args...)
+	var mdfindArgs = []string{"-literal", "kMDItemDisplayName == \"generate-bazel-extra\"", "kind:folder"}
```

Comment:
`mdfind` isn't a standard Unix tool, so it won't necessarily be installed everywhere we need it.

I was expecting you would find the location of the current binary by checking `os.Args[0]`. Does that not work for some reason?